server/command.go

```diff
@@ -70,6 +70,58 @@ func (p *Plugin) handleHelp(args []string, extra *model.CommandArgs) (string, er
 	return p.getHelpText(), nil
 }
 
+func (p *Plugin) handleStartConfirm(args []string, extra *model.CommandArgs) (string, error) {
+	if len(args) > 1 {
+		return tooManyParametersText, nil
+	}
+	userID := extra.UserId
+	user, appErr := p.API.GetUser(userID)
+	if appErr != nil {
+		return "Cannot get user.", errors.Wrap(appErr, "cannot get user")
+	}
+	channelID := extra.ChannelId
+	channel, err := p.API.GetChannel(channelID)
+	if err != nil {
+		return "", err
+	}
+	message := ""
+	if channel.IsGroupOrDirect() {
+		var members *model.ChannelMembers
+		members, err = p.API.GetChannelMembers(channelID, 0, 100)
+		if err != nil {
+			return "", err
+		}
+
+		if members != nil {
+			p.API.LogDebug(fmt.Sprintf("%d members in channel %s", len(*members), channelID))
+			membersCount := len(*members)
+			message += "\n" + fmt.Sprintf("You are about a create a meeting in a channel with %d members", membersCount)
+		}
+
+	}
+
+	post := &model.Post{
```

Comment:
Are we always showing the warning? Shouldn't we just show the warning when it is needed (i.e. when you are posting on a public or private channel)?